src/Runner.Worker/FileCommandManager.cs

```diff
@@ -266,7 +266,7 @@ public void ProcessCommand(IExecutionContext context, string filePath, Container
 
     public sealed class CreateStepSummaryCommand : RunnerService, IFileCommandExtension
     {
-        private const int _attachmentSizeLimit = 128 * 1024;
+        public const int AttachmentSizeLimit = 1024 * 1024;
```

Comment:
do you want to change this to pass down the limit from service? so you don't need to update the runner every time you need to bump the limit, especially if you need to bump the limit for certain customers.

Reply:
I don't want to manage limits per users if it can be helped at all... 😅 

Yeah passing it down from server may be a good choice if we find we need to do this again in the future.  Having the limit defined in a public repo is still a good communication tool IMHO though - we could point others to this number and they will understand the limit right away.  😄   However if we find we need to update this often, then yeah... 

Maybe next time we should visit this decision?

Reply:
I am fine with it.